ledger/src/shred/stats.rs

```diff
@@ -104,6 +108,22 @@ impl ProcessShredsStats {
             ("num_data_shreds_63", self.num_data_shreds_hist[3], i64),
```

Comment:
Should we maybe add comments about what each metric measures when we add new metrics?

Reply:
done